ext/bg/js/options.js

```diff
@@ -214,7 +214,9 @@ function optionsSetDefaults(options) {
             autoHideResults: false,
             delay: 20,
             length: 10,
-            modifier: 'shift'
+            modifier: 'shift',
+            popupNestingMaxDepth: 2,
+            enableOnSearchPage: true
```

Comment:
I think this should be off by default, because users who don't have Japanese dictionaries installed could find it very confusing if another popup appeared when moving their mouse on top of the expressions above the glossary. Maybe there should also be an option for disabling clicking the terms if the user likes to close the recursive popup by clicking on the page (popup one level below in this case). That has to be kept on by default, though, because there is already a stable release where it's always on. I might have preferred to keep it disabled by default, but too late now.

Reply:
Disabled by default.